src/applications/disability-benefits/wizard/pages/bdd.jsx

```diff
@@ -42,89 +37,81 @@ const findNextPage = state => {
   }
 
   if (differenceBetweenDatesInDays < 90) {
-    saveDischargeDate(dateDischarge, false);
+    saveDischargeDate(date, false);
     return pageNames.fileClaimEarly;
   }
 
   if (differenceBetweenDatesInDays <= 180) {
-    saveDischargeDate(dateDischarge, true);
+    saveDischargeDate(date, true);
     return pageNames.fileBDD;
   }
 
   saveDischargeDate();
   return pageNames.unableToFileBDD;
 };
 
-const defaultState = {
-  day: {
-    value: '',
-    dirty: false,
-  },
-  month: {
-    value: '',
-    dirty: false,
-  },
-  year: {
-    value: '',
-    dirty: false,
-  },
-};
-
 const label =
   'What’s the date or anticipated date of your release from active duty?';
 
-const isDateComplete = date =>
-  date.day.value && date.month.value && date.year.value.length === 4;
-
-const getDate = date =>
-  moment({
-    day: date.day.value,
-    month: parseInt(date.month.value, 10) - 1,
-    year: date.year.value,
-  });
-
+const getDate = date => moment(date, dateTemplate);
+const isDateComplete = date => date?.length === dateTemplate.length;
 const isDateInFuture = date => date?.diff(moment()) > 0;
 const isDateLessThanMax = date => date?.isBefore(maxDate);
 
-const BDDPage = ({ setPageState, state = defaultState }) => {
+const BDDPage = ({ setPageState, state = '' }) => {
   const [ariaDescribedby, setAriaDescribedby] = useState('');
+  const [isDirty, setIsDirty] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
since your setting it to `null` below, maybe the default state should also be `null`?

Reply:
Good point! Thanks!